lib/govuk_index/presenters/common_fields_presenter.rb

```diff
@@ -81,6 +81,18 @@ def section_id
       @_section_id ||= payload.dig("details", "section_id") if format == "hmrc_manual_section"
     end
 
+    def is_historic
```

Comment:
I wonder if this and is_political should be `historic?` and `political?`. This would also mean a change to `is_withdrawn` for consistency.

This would be more idiomatic to Ruby as prefixing methods with `is_`, `has_` and `can_` are [regularly discouraged](https://docs.rubocop.org/en/latest/cops_naming/#namingpredicatename). Doing a search in search-api codebase shows that the only 'def is_' method we have like this is `is_withdrawn`

Reply:
Good point.  I didn't include the `?` to maintain consistency with `is_withdrawn`, but it's definitely a better idea to update the name of `is_withdrawn` to `withdrawn?`.